lib/ecl/ecl_kw.c

```diff
@@ -1711,9 +1714,18 @@ void ecl_kw_fwrite_header(const ecl_kw_type *ecl_kw , fortio_type *fortio) {
 }
 
 
-void ecl_kw_fwrite(const ecl_kw_type *ecl_kw , fortio_type *fortio) {
-  ecl_kw_fwrite_header(ecl_kw ,  fortio);
-  ecl_kw_fwrite_data(ecl_kw   ,  fortio);
+bool ecl_kw_fwrite(const ecl_kw_type *ecl_kw , fortio_type *fortio) {
+  bool verify_kw_header_len = true;
```

Comment:
This `bool` variable is not necessary - this whole function can be simplified to something like:
```
if (strlen(ecl_kw_get_header( ecl_kw)) > ECL_STRING8_LENGTH) {
   fortio_fwrite_error(fortio);
   return false;
}

ecl_kw_fwrite_header(ecl_kw ,  fortio);
ecl_kw_fwrite_data(ecl_kw   ,  fortio);
return true;
```